ssclj/rpm/src/main/systemd/ssclj.service

```diff
@@ -1,13 +1,22 @@
 [Unit]
 Description=SlipStream API
-After=syslog.target network.target
+After=syslog.target
+After=network.target
```

Comment:
Yes, the "After" configurations are equivalent and I think that the one with separated values is clearer.